packages/bpmn-editor/src/svg/BpmnDiagramSvg.tsx

```diff
@@ -210,6 +210,7 @@ export function getNodeLabelSvgTextAlignmentProps(
         width: clWidth,
       } as const;
 
+    case "center-left-vertical":
```

Comment:
Ended up needing to add the new position to this SVG generation file to avoid errors. Not sure if there's a better practice than creating an empty case, but I saw a similar thing being done in the NodeStyle.ts above